src/components/SpruceForm/Widgets/LeafyGreenWidgets.tsx

```diff
@@ -45,11 +45,14 @@ export const LeafyGreenTextInput: React.VFC<
     optional,
     warnings,
   } = options;
+
   const hasError = !!rawErrors?.length;
+  const deduplicatedErrors = Array.from(new Set(rawErrors));
```

Comment:
For some reason I keep getting duplicated errors on the text inputs, again _only_ on the Project Settings page, which is why I added this... I'm not sure if this a RJSF bug or I'm doing something wrong?

<img width="479" alt="Screen Shot 2022-06-24 at 3 40 27 PM" src="https://user-images.githubusercontent.com/47064971/176028156-caf67b3a-9cf9-4ea2-8e9d-df8508171b2d.png">